handlers/api.go

```diff
@@ -4095,12 +4095,12 @@ func adjustQueryResults(data []interface{}, adjustQueryEntriesFuncs ...func(map[
 	return nil
 }
 
-// SendErrorResponse exposes sendErrorResponse
```

Comment:
**Minor:** This comment now is outdated. Also, search for `sendErrorResponse` within the sources and you will find two more locations. Those are both code that is commented out but I suggest to either remove it completely or to fix it.

Reply:
remove